src/main/java/ro/vladutit/Don/t/forget/v2/controller/ProfileController.java

```diff
@@ -0,0 +1,24 @@
+package ro.vladutit.Don.t.forget.v2.controller;
```

Comment:
wrong controller. the user profile class should be in the user's controller

Reply:
When I first started coding, I looked online for how to structure the project. I found it to be in two ways: 
- code organized by layer 
- code organized by feature. 

My project is organized by layer. 
So, for now, I had: controller, model, repository, service, etc. All the controllers I had in the project, are in the Controller folder.